src/desktop/apps/artwork/client/bootstrap.coffee

```diff
@@ -6,3 +6,4 @@ module.exports = (sd, { artwork }) ->
     href: artwork.href
     artists: artwork.artists
     pageviews: artwork.pageviews
+    is_acquireable: artwork.is_acquireable
```

Comment:
So this will exclude Make Offer-only works for now?

Reply:
Yep! We can add `is_offerable` as well for that case.